src/test/jtx/envconfig.h

```diff
@@ -27,8 +27,7 @@ namespace test {
 
 extern std::atomic<bool> envUseIPv4;
 
-inline
```

Comment:
I setup my editor to use the `.clang-format` file in the repo, but inadvertently reformatted code that I didn't touch. I'm going to revert those formatting changes to this file (and the others).

Reply:
I'll revert these changes when I squash the commits